openassessment/templates/openassessmentblock/response/oa_response.html

```diff
@@ -53,16 +62,49 @@ <h4 class="step__title">
         <div class="wrapper--step__content">
             <div class="step__instruction">
                 <p>
+                  {% if team_name %}
+                    {% trans "Enter your team's response to the prompt." %}
+                    {% if submission_due %}
+                      {% blocktrans %}
+                        You can save your progress and return to complete your team's response at any time before the due date
```

Comment:
missing period.

Reply:
Confusingly, there is a period but it's hiding after the "date" span that's after this.